.github/workflows/ci.yml

```diff
@@ -64,7 +62,24 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: pnpm
       - run: pnpm install --frozen-lockfile
-      - run: pnpm playwright install
+      - name: Set Environment Variables for Playwright (non-windows)
+        if: runner.os != 'Windows'
+        run: |
+          echo "PLAYWRIGHT_VERSION=`pnpm playwright --version`" >> $GITHUB_ENV
+          echo "PLAYWRIGHT_BROWSERS_PATH=$HOME/__playwight_browsers" >> $GITHUB_ENV
+      - name: Set Environment Variable of Playwright Browsers Path (windows)
+        if: runner.os == 'Windows'
+        run: |
+          $PLAYWRIGHT_VERSION = (pnpm playwright --version)
+          echo "PLAYWRIGHT_VERSION=$PLAYWRIGHT_VERSION" >> $env:GITHUB_ENV
+          echo "PLAYWRIGHT_BROWSERS_PATH=$HOME\__playwight_browsers" >> $env:GITHUB_ENV
+      - name: Cache Playwright
+        uses: actions/cache@v3
+        id: playwright_cache
+        with:
+          path: ${{ env.PLAYWRIGHT_BROWSERS_PATH }}
+          key: OS - ${{ runner.os }} - Node ${{ matrix.node-version }} - Browser ${{ matrix.e2e-browser }} - Playwright Version - ${{ env.PLAYWRIGHT_VERSION }}
```

Comment:
Is keying by node version required? i don't think playwright installs different browser versions depending on the node version used.

Reply:
The only place where this could go wrong is when two matrix jobs try to update the cache with the same key at the same time. :thinking: 

They would update with the same content though so either first-one-wins of last-one-wins. the winner should be picked up by subsequent runs. It would be bad if the loser failed it's run though

Reply:
OK, done it